xbmc/pvr/recordings/PVRRecordings.cpp

```diff
@@ -39,7 +39,10 @@ CPVRRecordings::CPVRRecordings(void) :
     m_bIsUpdating(false),
     m_iLastId(0),
     m_bGroupItems(true),
-    m_bHasDeleted(false)
+    m_bHasDeletedTV(false),
+    m_bHasDeletedRadio(false),
```

Comment:
`m_bDeletedTVItems` as we use `b` are prefix there is no need for the `Has`. And adding a suffix Items or Recordings would be good.